pkg/scheduler/service/service.go

```diff
@@ -68,6 +69,7 @@ const (
 	defaultReconcileInterval = 300    // 5m default task reconciliation interval
 	defaultReconcileCooldown = 15 * time.Second
 	defaultHttpBindInterval  = 5 * time.Second
```

Comment:
might want make the default http bind interval part of the scheduler fine tuning config file.